assets/js/components/data-block.js

```diff
@@ -71,15 +72,14 @@ class DataBlock extends Component {
 		} = this.props;
 
 		const role = ( 'button' === context ) ? 'button' : '';
-
```

Comment:
Any reason for removing this? It seems unrelated and I've always liked a newline separating out the `render()` logic from its return value/`render()` output.

I'd say this should be restored.

Reply:
This is removed already. But it seems github is not picking it up properly.